### DISCUSSION TITLE
Step by step get started?

### DISCUSSION THREAD

**Discussion by bartlanz** [toverainc/willow#73] [General]
Is there a step by step guide for getting started with this? 

Currently at my house I have an ESXi Server hosting Home Assistant and other things. I have a few of the ESP32 S3 Boxes. I have read through the Git hub, and I do not see a start here, do this type article.

**Comment by hamishcunningham** [comment]
I got mine running using the steps described in the top-level README: https://github.com/toverainc/willow/blob/main/README.md

**Comment by kristiankielhofner** [comment]
We will also start work soon on a one-shot install script but for now you will have to follow the README. I know it looks involved and daunting but when it really comes down to it it's something like 10 commands total.

With this initial release of Willow we wanted to make sure our users had a greater understanding of the process because we are advancing very rapidly and you can keep up with all of the new work we're doing much more easily if you can build and are familiar with the process.
